packages/angular/src/generators/init/init.ts

```diff
@@ -8,8 +8,9 @@ import {
   Tree,
   updateNxJson,
 } from '@nrwl/devkit';
-import { jestInitGenerator } from '@nrwl/jest';
+import { jestInitGenerator } from '@nrwl/jest/generators';
 import { Linter } from '@nrwl/linter';
+import { initGenerator as jsInitGenerator } from '@nrwl/js';
```

Comment:
Looking at previous PR review comments, I realize these changes might be needed because of TS imports from other files exported in plugin entry points. I know it might get big, but we try to address that instead? The issue with the inconsistencies above is that it is not straightforward to know from where I need to import a generator. It makes maintenance and contributions harder. Right now, it would seem that the best bet is to do a deep import to reduce the likelihood of an issue with TS imports. Otherwise, it's more like a trial and error type of thing.

Reply:
done. changed the import to just from the root package.